webrtc.html

```diff
@@ -3012,6 +3013,10 @@ <h2>Methods</h2>
                     </ol>
                   </li>
                 </ol>
+                <p class="note">In the above algorithm, addIceCandidate(null)
+                succeeds, indicating end-of-candidates for all media
```

Comment:
Weird sentence, should it say "use addIceCandidate(null) to indicate end-of-candidates" or "if addIceCandidate(null) succeeds..."? What does it mean to indicate end of ICE candidate generation in the context of adding?